libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/RustTimeline.kt

```diff
@@ -343,6 +350,7 @@ class RustTimeline(
     }
 
     @Throws
+    @Suppress("UnusedPrivateMember")
```

Comment:
I added this warning suppression here, but maybe it's better to just remove the function?

Reply:
Yes, maybe just remove it.